apps/design/frontend/src/contest_form.tsx

```diff
@@ -1,57 +1,129 @@
 import React, { useState } from 'react';
-import {
-  Button,
-  Table,
-  TH,
-  TD,
-  P,
-  SegmentedButton,
-  SearchSelect,
-  Modal,
-  Callout,
-} from '@votingworks/ui';
 import { useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+import { z } from 'zod/v4';
+
+import { Result, throwIllegalValue } from '@votingworks/basics';
 import {
+  ElectionId,
   AnyContest,
+  DistrictId,
+  PartyId,
   Candidate,
   CandidateContestSchema,
   CandidateId,
   ContestId,
-  DistrictId,
-  ElectionId,
-  PartyId,
   safeParse,
   YesNoContestSchema,
 } from '@votingworks/types';
-import { Result, throwIllegalValue } from '@votingworks/basics';
-import { z } from 'zod/v4';
-import { FieldName, Form, FormActionsRow, InputGroup, Row } from './layout';
-import { routes } from './routes';
 import {
-  createContest,
-  deleteContest,
+  Callout,
+  SearchSelect,
+  SegmentedButton,
+  P,
+  TH,
+  TD,
+  Button,
+  Modal,
+  DesktopPalette,
+  Table,
+} from '@votingworks/ui';
+
+import {
+  getBallotsFinalizedAt,
   getElectionInfo,
   listDistricts,
   listParties,
+  createContest,
   updateContest,
+  deleteContest,
 } from './api';
+import {
+  FormFixed,
+  FormBody,
+  FormErrorContainer,
+  FormFooter,
+} from './form_fixed';
+import { InputGroup, Row, FieldName } from './layout';
+import { routes } from './routes';
+import { TooltipContainer, Tooltip } from './tooltip';
 import { generateId, replaceAtIndex } from './utils';
 import { RichTextEditor } from './rich_text_editor';
 
+const Form = styled(FormFixed)`
+  .search-select,
+  input[type='text'] {
+    max-width: 20rem;
+    min-width: 5rem;
+    width: 100%;
+  }
+
+  .icon-button {
+    background: ${(p) => p.theme.colors.background};
+    border: ${(p) => p.theme.sizes.bordersRem.thin}rem solid
+      ${(p) => p.theme.colors.outline};
+    padding: 0.6rem 0.75rem;
+  }
```

Comment:
Are these CSS tweaks for the icon buttons necessary or can we just use the default styles?

Reply:
Just aesthetic - was going for something closer to square; the default padding without text looked slightly off to me, WRT the vertical/horizontal ratio, but open to change if the current look isn't preferable

Reply:
Cool, square sounds good. Maybe we can build those styles into the button component? Doesn't have to be in this PR